### PR TITLE
add description about formatting go code

### PR DESCRIPTION
**Pull request by kaakaa**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ REVISION := $(shell git rev-parse --short HEAD)
 LDFLAGS := -ldflags="-s -w -X \"main.Version=$(VERSION)\" -X \"main.Revision=$(REVISION)\" -extldflags \"-static\""
 DIST_DIRS := find * -type d -exec
 
-.PHONY: glide deps clean test cross-build dist
+.PHONY: glide deps clean check-style test coverage cross-build dist
 
 all: dist test
```
Comment (Hanzei) [inline Makefile]: Can we add `check-style`?

File: Makefile [hunk 2]
```diff
@@ -36,6 +36,17 @@ test:
 coverage: test
 	go tool cover -html=coverage.txt
 
+check-style:
+	@echo Running GOFMT
+	$(eval GOFMT_OUTPUT := $(shell gofmt -d -s poll/ 2>&1))
+	@echo "$(GOFMT_OUTPUT)"
```
Comment (Hanzei) [inline Makefile]: This produces a lot of output. Mind removing it?

File: Makefile [hunk 3]
```diff
@@ -36,6 +36,17 @@ test:
 coverage: test
 	go tool cover -html=coverage.txt
 
+check-style:
+	@echo Running GOFMT
```
Comment (Hanzei) [inline Makefile]: Later you use lower case. Can we choose one style?

File: Makefile [hunk 4]
```diff
@@ -36,6 +36,17 @@ test:
...
+	$(eval GOFMT_OUTPUT := $(shell gofmt -d -s poll/ 2>&1))
```
Comment (Hanzei) [inline Makefile]: Is it possible to run `gofmt` for every `.go` file?

Comment (kaakaa) [inline Makefile]: Now, this checks .go files only in `poll` dir.
I'll fix this.

File: Makefile [hunk 5]
```diff
@@ -37,13 +37,14 @@ coverage: test
...
-	@echo "$(GOFMT_OUTPUT)"
+	@echo Running gofmt
+	$(eval GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path './vendor/*'))
+	$(eval GOFMT_OUTPUT := $(shell gofmt -l -s $(GOFILES_NOVENDOR) 2>&1))
 	@if [ ! "$(GOFMT_OUTPUT)" ]; then \
 		echo "gofmt success"; \
 	else \
-		echo "gofmt failure"; \
+		echo "gofmt failure. You must format the following .go files."; \
+		echo "  $(GOFMT_OUTPUT)"; \
```
Comment (Hanzei) [inline Makefile]: It help the UX if we would provide an easy cody past command for the user like:
```
gofmt failure. Please run:
gofmt -w -s $(GOFMT_OUTPUT)
```

Comment (kaakaa) [inline Makefile]: That's good! I'll fix it.

**Comment by kaakaa** [comment]
Fixed the message.

**Pull request merged**